lua/nvim-treesitter/configs.lua

```diff
@@ -300,17 +300,22 @@ function M.setup(user_data)
       config.ensure_installed = data
       require'nvim-treesitter.install'.ensure_installed(data)
     else
+      data.disable = data.disable or {}
+      for _, lang in ipairs(user_data.disable or {}) do
+        if not vim.tbl_contains(data.disable, lang) then
+          table.insert(data.disable, lang)
+        end
+      end
       config.modules[name] = vim.tbl_deep_extend('force', config.modules[name] or {}, data)
 
       recurse_modules(function(_, _, new_path)
         if data.enable then
           enable_all(new_path)
         end
-
-        for _, lang in ipairs(data.disable or {}) do
+        for _, lang in ipairs(config.modules[name].disable) do
```

Comment:
Isn't this the old buggy version? Just tbl_merge it with the module disable in the new version.

@vigoux  We should remove markdown since it's only SEGFAULTing

Reply:
Or with @Konan11221 solution, we can even keep it for people who like to experiment. Could you extend the README with a note which parsers are disabled by default. Thank you for your contribution.